### ISSUE TITLE
[SUGGESTION] power cells should be shrunk

### ISSUE THREAD

**Issue by TIKIRobo** [tsunamayo/Starship-EVO#5778]
![Screenshot 2024-05-26 181630](https://github.com/tsunamayo/Starship-EVO/assets/56371294/f72a7c59-0c2b-4754-b674-391ecc0f55f4)
here are two mostly identical ships and their total flight times

the ship on the left (assuming its at full charge) can power two meters of thrusters (100% of its power) for 26.6 minutes 
the ship on the right (assuming full fuel) can power 4.5 m of thrusters for 40 minutes

both power sources take up the same area but the reactor is just plain better, however if the battery was shrunk down that would help with a lot of things 
first this battery is meant to be portable and used in small hovers/ships much like an IRL car battery   
![image](https://github.com/tsunamayo/Starship-EVO/assets/56371294/a11904a7-7444-432c-8d0a-33908a0c2543)
instead its almost the size of the player's torso 
![image](https://github.com/tsunamayo/Starship-EVO/assets/56371294/c1372b6b-85bd-43f8-b068-86481c3634e0)
it does not fit small craft
second it would make the lower power output more palette-able as the smaller size would match its current output

TLDR: the current battery system works well and the stats make sense but not for a battery that large

**Comment by ProPeach** [comment]
Definitely agree on reducing the size of the Power Cell, something closer to that Capacitor would be great. Its just too big when combined with the charging base to incorporate onto a lot of starter hovercraft.


Also, I think it'd be a shame if these early game bricks *only* made sense in the early game and were completely overshadowed in the long run by later technologies. Ideally these Cells would still be a viable power source throughout the game as it will provide the player with choices to make and design around rather than just If Fuel Unlocked -> Only Use Fuel

**Comment by Gallt** [comment]
100% agree on both points.

**Comment by tsunamayo** [comment]
Yeah I hesitated to make it quarter meter, I will add a smaller one. Btw I didnt really intended to make it for spaceship, but for hovercraft and other smaller land vehicles. But I guess for a smaller spaceship it could work, just dont expect a lot of uptime with just a single cell.
